x-pack/legacy/plugins/apm/typings/es_schemas/raw/ErrorRaw.ts

```diff
@@ -27,6 +27,7 @@ interface Exception {
   module?: string;
   handled?: boolean;
   stacktrace?: IStackframe[];
+  [key: string]: unknown;
```

Comment:
Is this still necessary after you updated `AllowUnknownProperties` to support arrays?

Reply:
(I removed it and didn't get any warnings so I assume it's not needed)